srcpkgs/proton-bridge/patches/init.sh.patch

```diff
@@ -0,0 +1,4 @@
+5c5
```

Comment:
Not specified which file to patch. did you created this file with `diff oldfile newfile`?

Reply:
try `-u` then, from wrksrc level directory